modin/dataframe/test/test_groupby.py

```diff
@@ -207,7 +207,7 @@ def test_large_row_groupby():
 
     ray_df = from_pandas(pandas_df, 2)
 
-    by = pandas_df['A'].tolist()
+    by = [str(i) for i in pandas_df['A'].tolist()]
```

Comment:
This change is necessary because we have column names that are integers, and the result of `pandas_df['A']` will be integers. It will follow the codepath where it tries to group by every column.